howtos/configuration.md

```diff
@@ -18,15 +18,6 @@ Our Algolia API key, app ID, and index name are configured in `./docusaurus.conf
 
 Configuration of Algolia's index and crawler can be edited with the [Algolia Crawler](https://crawler.algolia.com/).
 
-Currently, the only unique change to our configuration is that we specify multiple `pathsToMatch` to accommodate our multiple docs instances:
```

Comment:
I also changed this setting in our Algolia crawler config, to remove the `optimize` line.